### ISSUE TITLE
Forge Server

### ISSUE THREAD

**Issue by ImJustPax** [BookBackdoor/BookBackdoor#6]
it says any minecraft server so does a Forge server count aswell?

**Comment by desktopdotini** [comment]
No.

**Comment by crypticplank** [comment]
No, Currently this only supports PaperMC Servers. A new backdoor is being worked on, and will be released shortly.

I may plan on adding support for fabric, forge, spigot and bukkit later.
